### ISSUE TITLE
NTR: Galactokinase activity low in erythrocytes (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8217]
New term request Galactokinase activity low in erythrocytes (elthat_100806151824): GALK; RBCs, red blood cells  UMLS: 2673555

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000723
- label: Reduced erythrocyte aglactokinase activity
- parents: [Abnormal erythrocyte enzyme concentration or activity (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:7670469
- def: Activity or concentration of in the level of galactokinase (EC 2.7.1.6) in erythrocytes below the lower limit of normal.
- comment: Galactokinase catalyzes the phosphorylation of galactose to galactose-1-phosphate, the first step in galactose metabolism.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8217
- synonyms:
